lambda-code/notify-slack/src/utils.ts

```diff
@@ -1,3 +1,8 @@
+import AWSXRay from "aws-xray-sdk-core";
+// Need to wrap the http module globally before importing it
+// to ensure that all outgoing HTTP requests are captured by AWS X-Ray.
+AWSXRay.captureHTTPsGlobal(require("http"));
+
```

Comment:
Using `require` causes a compatibility issue because our Lambda functions are set up with ESM.

Here is one quick solution I found online (there may be other ones though). I already tested it:
```
import AWSXRay from "aws-xray-sdk-core";

(async () => {
  const http = await import("http");
  const https = await import("https");

  // These are CommonJS modules, so use .default if needed
  AWSXRay.captureHTTPsGlobal(http.default || http);
  AWSXRay.captureHTTPsGlobal(https.default || https);
})();
```

In this Lambda function I see that we are importing `https` so we should be tracking that specific package.

Reply:
I have a plan to redo this PR and not use the aws xray sdk but instead use the opentelemetry directly that allows for more configuration and compatability long term.